src/PluginClient.php

```diff
@@ -62,6 +62,36 @@ public function __construct($client, array $plugins = [], array $options = [])
     }
 
     /**
+     * Adds a plugin to the client.
+     *
+     * @param Plugin $plugin
+     */
+    public function addPlugin(Plugin $plugin)
```

Comment:
I don't really think it matters, both sounds OK. The reason why I usually prefer add is that it fits into the three character long actions: add, get, set, has